diff --git a/server/ktamv_server_dm.py b/server/ktamv_server_dm.py
index 1e5a964..5a1c4b9 100644
--- a/server/ktamv_server_dm.py
+++ b/server/ktamv_server_dm.py
@@ -247,7 +247,7 @@ class Ktamv_Server_Detection_Manager:
                 # use the one closest to the center of the image.
                 closest_index = self.find_closest_keypoint(keypoints)
                 # create center object from centermost keypoint
-                (x,y) = np.around([keypoints[closest_index]].pt)
+                (x,y) = np.around(keypoints[closest_index].pt)
             else:
                 # create center object from first and only keypoint
                 (x,y) = np.around(keypoints[0].pt)
@@ -303,4 +303,4 @@ class Ktamv_Server_Detection_Manager:
-    def find_closest_keypoint(keypoints):
+    def find_closest_keypoint(self, keypoints):
         closest_index = None
         closest_distance = float('inf')
         target_point = np.array([320, 240])
